base/src/com/google/idea/blaze/base/sync/SyncPhaseCoordinator.java

```diff
@@ -360,8 +360,23 @@ void runSync(BlazeSyncParams params, boolean singleThreaded, BlazeContext contex
           ProjectStateSyncTask.collectProjectState(project, params.blazeBuildParams(), context);
       BlazeSyncBuildResult buildResult =
           projectState != null
-              ? BuildPhaseSyncTask.runBuildPhase(project, params, projectState, buildId, context)
+              ? BuildPhaseSyncTask.runBuildPhase(project, params, projectState, buildId, context,false)
```

Comment:
Couldn't you utilize the `BlazeSyncParams`/`BlazeContext ` to contain the `boolean` field?
They both available at the `runBuildForTargets(...)` method signature.

Reply:
So BlazeSyncParams isn't there but BlazzeBuildParams is. I was able to work around that. Take a look.
Thanks for the suggestion!